includes/admin/class-wc-stripe-admin.php

```diff
@@ -0,0 +1,71 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+
+/**
+ * Class that handles various admin tasks.
+ *
+ * @since 5.6.0
+ */
+class WC_Stripe_Admin {
```

Comment:
The name `WC_Stripe_Admin` seems a bit generic for something whose purpose if of enqueuing the script for the payments tab.
What do you think about renaming this to something less generic? Maybe `WC_Stripe_Payment_Gateways_Controller` (so that it matches the name of the asset that it enqueues) or `WC_Stripe_Payments_Page_Controller` (or something similar - I'm open to other possibilities)?

Reply:
Agreed - I like `WC_Stripe_Payment_Gateways_Controller`.